capitals.rb

```diff
@@ -150,4 +153,50 @@
 }, {
     name: "Wyoming",
     capital: "Cheyenne"
-}]
+}].shuffle
+
+# states arranged in non alphabetical order
+
+
+# Initialize new keys to sort the num time of correct and wrong answers
+states[:num_correct]
+states[:num_wrong]
+user_input = []
+index = 0
+
+# prompt the user to name the capital of each 50 state
+ # index = 0
+ # loop do
+ #   # make prompt appear
+ #   puts "What is the capital of " + state
+ #      user_input =gets.chomp.to_s
+ #      if user_input === capital <<
+ #        index +=1
+ #      end
+ #      if user_input == num_wrong
+ #        puts "WRONG"
+ #      end
+ #   break if index == states.length
+ # end
+
+
+states.each do |name|
+  puts "What is the capital of"  + name[:name]
```

Comment:
since `states` is an array of Hashes that represent states, make your iterator variable `state` instead of `name`